NSS-Spontaneous-Spoonbills/NSS-Spontaneous-Spoonbills
# Created a customer/seller with REST

## Description of Proposed Changes

## Steps to Test

Outline the steps to test
1. Setup virtual machine
1. Run server 
1 Go to http://127.0.0.1:8000/bangazonapi/
1. Check and make sure url works
1. Click on link http://127.0.0.1:8000/bangazonapi/user/
1. Create user with REST framework
1. Adding walkthrough to README.md
```
git fetch --all
git checkout JS_creating_rest_framework
```

## Impacted Areas in Application
List general components of the application that this PR will affect:
##### Modified
bangazon/settings.py
bangazon/urls.py
##### Deleted
bangazonapi/models.py
bangazonapi/views.py
##### Created
bangazonapi/serializers/
bangazonapi/urls.py
views/

## Mentions @username

Tag users that need to review this code
@MrErin 
@obscure-shadow 


## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
1. Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
